OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -19,47 +19,165 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class SupportPowerTimerWidget : Widget
 	{
+		public readonly int XObserver = 10;
```

Comment:
Widgets shouldn't define their own conditional offsets like this. It is the job of the UI yaml or logic plumbing to move them.